apps/smartling/lambda/src/index.ts

```diff
@@ -121,6 +121,18 @@ export function makeApp(fetchFn: any, issuer: any) {
     );
   });
 
+  const computeLastModifiedTime = () => {
```

Comment:
We were having some issues with the cache not being invalidated when deploying and copied this from Typeform's setup.